ui/components/component-library/input/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Input } from './input';
+export { InputType } from './input.types';
```

Comment:
Need a line break here
```suggestion
export { InputType } from './input.types';

```